packages/core/upload/admin/src/components/TableList/TableRows.js

```diff
@@ -51,25 +64,26 @@ export const TableRows = ({ assets, onEditAsset, onSelectAsset, selectedAssets }
               <Typography>{name}</Typography>
             </Td>
             <Td>
-              <Typography>{getFileExtension(ext).toUpperCase()}</Typography>
+              <Typography>{ext ? getFileExtension(ext).toUpperCase() : '-'}</Typography>
```

Comment:
Could we please centralize the rendering of `-` somewhere? In the CM we have a component (`CellContent`) which takes care of that. This allows us to change the fallback value in one place.